File: Fundamentos de Programação/Lista 6/Exe5.c
#include<stdio.h>

#define TAM 5  // ALTERAR PARA 10

int main()
{
    int vetor[TAM];
    int i, quadrado=0, soma_quadrado=0;

    for(i=0; i<TAM; i++)
    {
        printf("Digite o %do valor: ", i+1);
        scanf("%d", &vetor[i]);

        quadrado = vetor[i]*vetor[i];

        soma_quadrado = soma_quadrado+ quadrado;
    }

    printf("\n\n");

    printf("A soma dos quadrados dos valores digitados e %d\n\n", soma_quadrado);

    return 0;
}

File: Programação Orientada a Objetos/Lista 4/Exe5.java
public class Exe5{
	
	public static void main(String[] args) {

		int vetor[] = new int[10];
		int i;

		for(i=0; i<vetor.length; i++){

			vetor[i] = i+i;
		}

		for(int v : vetor){

			System.out.println(v);
		}
	}
}
